install-sh

```diff
@@ -0,0 +1,527 @@
+#!/bin/sh
```

Comment:
why include this whole file?  it should be brought automatically by `autoreconf`